### ISSUE TITLE
CLASS_NAMES in Argoverse

### ISSUE THREAD

**Issue by adricarda** [happinesslz/LION#13]
Hi thank you for releasing code and checkpoints for Argoverse. This is really useful for the community.
I am running your model in Argoverse but the output classes seems to be different to compared to the one reported in:
tools/cfgs/lion_models/lion_mamba_1f_1x_argo_128dim_sparse_v2.yaml.

For examples, all the detections that are clearly belonging to class "Regular_vehicle" should have label 0, while they are predicted as label 1 ("Pedestrian").  Is it possible that the classes are shifted by one?

Thank you

**Comment by AlmoonYsl** [comment]
> Hi thank you for releasing code and checkpoints for Argoverse. This is really useful for the community. I am running your model in Argoverse but the output classes seems to be different to compared to the one reported in: tools/cfgs/lion_models/lion_mamba_1f_1x_argo_128dim_sparse_v2.yaml.
> 
> For examples, all the detections that are clearly belonging to class "Regular_vehicle" should have label 0, while they are predicted as label 1 ("Pedestrian"). Is it possible that the classes are shifted by one?
> 
> Thank you

Hi, please refer to the code of voxelnext head, which is used for  Argoverse. The class_id_mapping_each_head is used for mapping the prediction labels of each head to origin labels. https://github.com/happinesslz/LION/blob/d6e068eadbda05a6e4beb01bf0dda361fcf0279e/pcdet/models/dense_heads/voxelnext_head.py#L459

**Comment by adricarda** [comment]
Thank you for your quick reply.
Apparently there is indeed a +1 on the predicted labels:

[            ret_dict[bidx]['pred_labels'] = torch.cat(ret_dict[bidx]['pred_labels'], dim=0) + 1](https://github.com/happinesslz/LION/blob/d6e068eadbda05a6e4beb01bf0dda361fcf0279e/pcdet/models/dense_heads/voxelnext_head.py#L486)

**Comment by AlmoonYsl** [comment]
> Thank you for your quick reply. Apparently there is indeed a +1 on the predicted labels:
> 
> [ ret_dict[bidx]['pred_labels'] = torch.cat(ret_dict[bidx]['pred_labels'], dim=0) + 1](https://github.com/happinesslz/LION/blob/d6e068eadbda05a6e4beb01bf0dda361fcf0279e/pcdet/models/dense_heads/voxelnext_head.py#L486)

Yes, be careful about whether the class labels start from 0 or 1.
